src/Tools/Source/CompilerGeneratorTools/Source/CSharpErrorFactsGenerator/CSharpErrorFactsGenerator.csproj

```diff
@@ -2,13 +2,24 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Almost a copy-paste from https://github.com/dotnet/roslyn/commit/875e10444fad7ad52adc5544081b2be6b939191a :)

Only difference is that I needed `Microsoft.CodeAnalysis.CSharp` instead of `Microsoft.CodeAnalysis.Common`